storage.go

```diff
@@ -28,6 +104,11 @@ func (s *Storage) delete(ctx context.Context, path string, opt pairStorageDelete
 
 	_, err = s.client.DeleteV2(input)
 	if err != nil {
+		if deleteErr, ok := err.(files.DeleteAPIError); ok {
+			if deleteErr.EndpointError.PathLookup.Tag == files.LookupErrorNotFound {
```

Comment:
Check `error_summary` could be more reliable.

Their response looks like following:

```http
{"error_summary": "path_lookup/not_found/..", "error": {".tag": "path_lookup", "path_lookup": {".tag": "not_found"}}}
```

We can check whether or not `error_summary` contains "path_lookup/not_found" like we did in `formatError`

Reply:
Or we can check if `err.Error()` contains `path_lookup/not_found` here.

Dropbox's error layout looks like:

```go
type DeleteV2APIError struct {
	dropbox.APIError
	EndpointError *DeleteError `json:"error"`
}

type APIError struct {
	ErrorSummary string `json:"error_summary"`
}

func (e APIError) Error() string {
	return e.ErrorSummary
}
```

So we can use `err.Error()` to get the content of internal APIError.